app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def links_count
+    Link.all.count
```

Comment:
@NishantUpadhyay-BTC this will fetch number of all links of the system. User should be able to see number of links he saved.